localisation/00_DEwc_united_localisation.csv

```diff
@@ -66,4 +66,5 @@ EVTDESCHFPB21098;Perhaps it is time I asked [From.GetTitledFirstName] to spend s
 EVTOPT_HFP_11011_B;Wait! How about you just take this [equipped_artifact.GetName] and let me live?!;Attendez�! Et si vous preniez ��[equipped_artifact.GetName]�� et me laissiez en vie�?;Nehmt [equipped_artifact.GetName] und lasst mich leben!;;�Alto! �Y si os llev�is este/a [equipped_artifact.GetName] y me dej�is vivir?;;;;;;;;;x
 EVTOPTA_RIP_29004;I will cherish it until the day I die!;Je vais le ch�rir jusqu'� ma mort�!;Ich werde [rip_dead.Getihnsie] in Ehren halten, bis zu meinem letzten Atemzug!;;�Lo cuidar� hasta el d�a que muera!;;;;;;;x;;
-EVTDESC_MNM_7057_NEW;The use of dark magic has corrupted me! The dark tendrils of [Root.GetDWDevil]'s magic has invaded my body, eroding mind and muscle alike. It is obvious that [Root.GetDWDevil] is keeping me alive, but at what cost?;L'utilisation de la magie noire m'a corrompu[Root.Get_E]�! Les vrilles mal�fiques de la magie [Root.GetDWDevilDuDela] ont envahi mon corps, �rodant � la fois mon esprit et mes muscles. Il est �vident que [Root.GetDWDevil] me maintient en vie, mais � quel prix�?;Der Gebrauch dunkler Magie hat mich verdorben! Die dunklen Ranken von [Root.GetDWDevil]s Magie sind in meinen K�rper eingedrungen und haben meinen Geist und meine Muskeln gleicherma�en beeinflusst. Es ist offensichtlich, dass [Root.GetDWDevil] mich am Leben h�lt, aber zu welchem Preis!?;;�Usar la magia oscura me ha corrompido! Los tent�culos oscuros de la magia de [Root.GetDWDevil] han invadido mi cuerpo, minando cuerpo y m�sculo por igual. Es obvio que [Root.GetDWDevil] me mantiene con vida, �pero a qu� precio?;;;;;;;;;x
+generic_artifact_prefix_de_r;;;r;;;;;;;;;;;x
+generic_artifact_prefix_de_s;;;s;;;;;;;;;;;xEVTDESC_MNM_7057_NEW;The use of dark magic has corrupted me! The dark tendrils of [Root.GetDWDevil]'s magic has invaded my body, eroding mind and muscle alike. It is obvious that [Root.GetDWDevil] is keeping me alive, but at what cost?;L'utilisation de la magie noire m'a corrompu[Root.Get_E]�! Les vrilles mal�fiques de la magie [Root.GetDWDevilDuDela] ont envahi mon corps, �rodant � la fois mon esprit et mes muscles. Il est �vident que [Root.GetDWDevil] me maintient en vie, mais � quel prix�?;Der Gebrauch dunkler Magie hat mich verdorben! Die dunklen Ranken von [Root.GetDWDevil]s Magie sind in meinen K�rper eingedrungen und haben meinen Geist und meine Muskeln gleicherma�en beeinflusst. Es ist offensichtlich, dass [Root.GetDWDevil] mich am Leben h�lt, aber zu welchem Preis!?;;�Usar la magia oscura me ha corrompido! Los tent�culos oscuros de la magia de [Root.GetDWDevil] han invadido mi cuerpo, minando cuerpo y m�sculo por igual. Es obvio que [Root.GetDWDevil] me mantiene con vida, �pero a qu� precio?;;;;;;;;;x
```

Comment:
Something went wrong here?

Reply:
seems like it, but I could easily correct this. done in next commit

Reply:
Just hope there is no more errors like this